### PR TITLE
BD-2107 Enable Verbose Logging in Unity editor

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes [#BD-2107](https://jira.braze.com/browse/BD-2107)

### CODE REVIEW

**Comment by rachel-feinberg** [comment]
@lydia-xie are checkboxes bolded or enclosed in quotes? I bolded them in this PR, but I've seen quotes in docs. I searched the style guides but couldn't find an explanation. Maybe I missed it.

**Comment by lydia-xie** [comment]
@rachel-feinberg - good question! the copy for a checkbox should be bolded - check out https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub#:~:text=Special%20Formatting.-,Describing%20Interactions%20with%20the%20UI,-When%20referring%20to (it's a long link lol but it's the "Describing Interactions with the UI" section)

**Comment by lydia-xie** [comment]
@cla-bot check

File: _docs/_developer_guide/platform_integration_guides/unity/sdk_integration/android.md [hunk 1]
```diff
@@ -147,6 +147,13 @@ Braze should now be collecting data from your application, and your basic integr
 
 ## Additional advanced implementation options
 
+### Enabling verbose logging in the Unity editor
+To enable verbose logging in the Unity Editor, do the following:
+
+1. Open the Braze Configuration Settings by navigating to **Braze > Braze Configuration**.
```
Comment (lydia-xie) [inline _docs/_developer_guide/platform_integration_guides/unity/sdk_integration/android.md:153]:
```suggestion
1. Open the Braze Configuration Settings by navigating to **Braze** > **Braze Configuration**.
```

File: _docs/_developer_guide/platform_integration_guides/unity/sdk_integration/ios.md [hunk 2]
```diff
@@ -56,6 +56,13 @@ Braze should now be collecting data from your application, and your basic integr
 
 ## Additional advanced implementation options
 
+### Enabling verbose logging in the Unity editor
+To enable verbose logging in the Unity Editor, do the following:
+
+1. Open the Braze Configuration Settings by navigating to **Braze > Braze Configuration**.
```
Comment (lydia-xie) [inline _docs/_developer_guide/platform_integration_guides/unity/sdk_integration/ios.md:62]:
```suggestion
1. Open the Braze Configuration Settings by navigating to **Braze** > **Braze Configuration**.
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
looks good - caught two small style edits!

**Review by lydia-xie — APPROVED** [review]

**Pull request merged**